.pre-commit-config.yaml

```diff
@@ -13,16 +13,13 @@ repos:
       - id: end-of-file-fixer
       - id: check-added-large-files
 
-  - repo: https://github.com/psf/black
-    rev: 23.12.1
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.4.3
     hooks:
-      - id: black
-        name: "Black: The uncompromising Python code formatter"
-        args: ["--line-length", "99"]
-
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.13.2
-    hooks:
-      - id: isort
-        name: "Sort Imports"
-        args: ["--profile", "black", "--py", "310"]
```

Comment:
Is `isort` missing in the new definition?

https://docs.astral.sh/ruff/formatter/#sorting-imports

Reply:
not needed as well